mentions-generator/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>mini-analytics</artifactId>
+        <groupId>de.brandwatch.minianalytics</groupId>
+        <version>0.1</version>
+    </parent>
+
+    <groupId>de.brandwatch.minianalytics</groupId>
+    <artifactId>mentions-generator</artifactId>
+    <version>0.1</version>
+
+    <properties>
+        <java.version>1.8</java.version>
```

Comment:
This only needs to be in the parent pom (where it currently doesn't seem to be).

Also, feel free to use a higher version if you want to. You can also stick with this one (it's what most of our services are still on).